src/storage/src/table/mem_table.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#![allow(dead_code)]
+#![allow(unused)]
+use std::collections::HashMap;
+
+use risingwave_common::array::Row;
+
+use crate::error::StorageResult;
+
+pub enum RowOp {
+    Insert(Row),
+    Delete(Row),
+    Update((Row, Row)),
+}
```

Comment:
By the way, there will be a `Record` enum like this in the future. We may consider reusing it at that time.

Reply:
suggest checking their common demand and do refactor in the future. the and here as a common interface of storage we just use `Row`, and `RowRef` in Chunk is different.

Reply:
btw, if we can implement this in `crate common`? (Though I do not see any usage of the abstraction.
 ```rust
enum Opreation<T> {
    Insert(T),
    Delete(T),
    Update((T, T)),
}
```

Reply:
@BugenZhao